suse_caasp

```diff
@@ -349,3 +349,20 @@ if check_rpm cri-o && [ -e /usr/bin/crictl ] && [ -S /var/run/crio/crio.sock ];
     log_cmd $OF 'journalctl -u crio'
 fi
 plugin_message Done
+
+#############################################################
+section_header "Checking gpu information ..."
+OF=gpu.txt
+plugin_message "All data stored in $OF"
+
+if [ -x /usr/bin/nvidia-smi ]
+then
+    log_cmd $OF 'nvidia-smi -q'
+    if [ -x /usr/bin/nvidia-container-cli ] 
+    then
+        log_cmd $OF 'nvidia-container-cli -k -d /dev/stderr --user=root:video info'
```

Comment:
I think the path to nvidia-smi and nvidia-container-cli might make sense to be in a variable. Then that variable should be tested for executability, and used again when the command is run - so the command run is definitely what's tested (rather than depending on $PATH here).  That means swapping the quotes around to allow the variable to be expanded.

```suggestion
NVIDIA_SMI="/usr/bin/nvidia-smi"
NVIDIA_CONTAINER_CLI="/usr/bin/nvidia-container-cli"
if [ -x "$NVIDIA_SMI" ]
then
    log_cmd "$OF" "$NVIDIA_SMI -q"
    if [ -x "$NVIDIA_CONTAINER_CLI" ] 
    then
        log_cmd "$OF" "$NVIDIA_CONTAINER_CLI" -k -d /dev/stderr --user=root:video info"
```